### ISSUE TITLE
Log / print filename of the key whose password is being requested

### ISSUE THREAD

**Issue by vladimir-v-diaz** [theupdateframework/tuf#568]
Note: The following issue has been reported by @trishankatdatadog and drafted by @vladimir-v-diaz.

**Description of issue or feature request**:

A user of the repository tool is often not given adequate, or, in certain instances, any information after executing an action.  Indication on what is happening is indeed logged, but isn't enabled on the console by default.  Ideally, output should also be colored or emphasized to improve readability.

**Current behavior**:

For example,

```
>>> private_ed25519_key = import_ed25519_privatekey_from_file('keystore/my_key')
```
*Enter a password for the encrypted Ed25519 key:*

**Expected behavior**:

```Python
>>> private_ed25519_key = import_ed25519_privatekey_from_file('keystore/my_key')
```
*Enter a password for **my_key**:*

**Comment by trishankatdatadog** [comment]
Could we update title on this, please? It should be something like: "Log / print filename of the encrypted key whose password is being requested"

**Comment by vladimir-v-diaz** [comment]
Okay, title has been updated.

**Comment by trishankatdatadog** [comment]
Thanks, Vlad, you're the best!

**Comment by vladimir-v-diaz** [comment]
This issue has been solved via upgrade to `securesystemslib` 0.10.9.
https://github.com/theupdateframework/tuf/pull/605
